public/docs/ts/latest/glossary.jade

```diff
@@ -158,10 +156,10 @@ block includes
     to a [View](#view) and handling most of the view’s display
     and user-interaction logic.
 
-    The Component is one of the most important building blocks in the Angular system.
+    The *Component* is one of the most important building blocks in the Angular system.
     It is, in fact, an Angular [Directive](#directive) with a companion [Template](#template).
 
-    The  developer applies the `!{_at}Component` !{_decoratorLink} to
+    You apply the `!{_at}Component` !{_decoratorLink} to
```

Comment:
This paragraph is correctly capitalized!

Except perhaps for "component instance", but I think that's just a matter of using "instance" in a different way than we usually do.